src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] dataRows;
+        try {
+            dataRows = getDataRows(fromFileName);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file",e);
+        }
```

Comment:
It is better to move try-catch to this method getDataRows()
```suggestion
        try {
            dataRows = getDataRows(fromFileName);
        } catch (IOException e) {
            throw new RuntimeException("Can't read file: " + fromFileName, e);
        }
```